precompiles/proxy/Proxy.sol

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity >=0.8.0;
+
+/// @author The Moonbeam Team
+/// @title The interface through which solidity contracts will interact with the Proxy pallet
+/// @custom:address 0x000000000000000000000000000000000000080b
```

Comment:
let's provide 2 constants similar too:
```
PROXY_ADDRESS = 0x000000000000000000000000000000000000080b

PROXY_CONTRACT = Proxy(0x000000000000000000000000000000000000080b)
```

(@nbaztec to see compare with popular contract and if name is consistent)

Reply:
(This needs to be done in other precompiles) maybe another PR to do that

Reply:
I was able to achieve this. Will submit a different PR with these exports.